Calibration/HcalCalibAlgos/test/analysis_sim_minbias_cfg.py

```diff
@@ -4,8 +4,8 @@
 
 # import of standard configurations
 process.load('Configuration.StandardSequences.Services_cff')
-process.load('Configuration.StandardSequences.GeometryExtended_cff')
-process.load('Configuration.StandardSequences.MagneticField_38T_cff')
+process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
+process.load('Configuration.StandardSequences.MagneticField_cff')
 process.load('Configuration.StandardSequences.RawToDigi_Data_cff')
 process.load('Configuration.StandardSequences.EndOfProcess_cff')
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
```

Comment:
Concerning https://github.com/cms-sw/cmssw/pull/8943/files#diff-139d4a3e6551d40d7eb5d611fdbfd1aeR34 (lines 34-62):

1. the input event data are collisions 2012C (RAW from HCAL NZS stream)
1. the Global Tag is for Run1 simulations
1. there is an `ESPrefer` customisation for loading data conditions.

I'd suggest to use `run1_data` key and remove the GT customisation.